README.md

```diff
@@ -310,3 +314,10 @@ To override this, you would have to redefine the `%_target_cpu` system macro (or
 Here is a demo showcasing the `Specfile.update_tag()` method and its use cases:
 
 [![Demo of Specfile.update_tag() functionality](https://img.youtube.com/vi/yzMfBPdFXZY/0.jpg)](https://www.youtube.com/watch?v=yzMfBPdFXZY)
+
+[badge-build]: https://softwarefactory-project.io/zuul/api/tenant/packit-service/badge?project=packit/specfile&pipeline=check
+[build]: https://softwarefactory-project.io/zuul/t/packit-service/buildsets?project=packit%2Fspecfile&pipeline=check&skip=0
+[docs-badge]: https://img.shields.io/badge/documentation-3572A5
+[docs]: https://packit.dev/specfile/api/specfile
+[badge-license]: https://img.shields.io/github/license/packit/specfile.svg
+[license]: ./LICENSE
```

Comment:
Hi!
Thanks for adding the links. The Github preview doesn't show these. Could you investigate further?
Thank you.

Reply:
Hi, 


I can see them just fine: https://github.com/jessp01/specfile/blob/47416e29c1e75daaca1089e606707ef3b2508629/README.md